tips-for-no-uppercase-letters-in-fulltext-index-name- (#1687)
vesoft-inc/nebula-docs: docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/1.text-based-index-restrictions.md

- Currently, full-text search supports `LOOKUP` statements only.

> - No uppercase letters are contained in a full-text index name.

Reviewer: ```suggestion
- Uppercase letters are not allowed in a full-text index name.
```
